Monika After Story/game/chess.rpy

```diff
@@ -179,7 +179,9 @@ init 1 python in mas_chess:
         "Tch-",
         "Winning isn't everything, you know...",
         "Ahaha,{w} I was just letting you win since you keep losing so much.",
-        "Oh, you won.{w} I should have taken this game seriously, then."
+        "Oh, you won.{w} I should have taken this game seriously, then.",
+        "I was just easy on you.",
```

Comment:
I really don't think we want to trivialize the player's chess victories against Monika any further.